tasks/go.py

```diff
@@ -354,6 +354,24 @@ def reset(ctx):
     ctx.run("rm -rf ./vendor")
 
 
+@task
+def check_go_mod_replaces(_ctx):
+    errors_found = set()
+    for mod in DEFAULT_MODULES.values():
+        go_sum = os.path.join(mod.full_path(), "go.sum")
+        if not os.path.exists(go_sum):
+            continue
+        with open(go_sum) as f:
+            for line in f:
+                if "github.com/DataDog/datadog-agent" in line:
```

Comment:
Could it be necessary to check case insensitive? In some context `DataDog` is `datadog` but here I don't know.

Reply:
I don't think it can be the case, but won't hurt checking against

Reply:
fixed in last commit

Reply:
It's the Github org name here so it should definitely be uppercased, but it doesn't cost much 😄